src/org/omegat/core/segmentation/SRX.java

```diff
@@ -418,24 +397,29 @@ public List<Exception> getExceptionsList() {
     private static final String DEFAULT_RULES_PATTERN = ".*";
 
     public static SRX getDefault() {
-        SRX srx = loadSrxFile(SRX.class.getResource("defaultRules.srx"));
-        srx.includeEndingTags = true;
-        srx.segmentSubflows = true;
-        return srx;
+        try {
+            SRX srx = loadSrxInputStream(SRX.class.getResourceAsStream("defaultRules.srx"));
+            srx.includeEndingTags = true;
+            srx.segmentSubflows = true;
+            return srx;
+        } catch (IOException e) {
```

Comment:
Is it OK for default values?

Reply:
Yes, for the default values you can refer to https://www.unicode.org/uli/pas/srx/srx20.html#formathandle